doc/usr/source/9_other/11_minimal_correction_set.rst

```diff
@@ -0,0 +1,65 @@
+.. _9_other/11_minimal_correction_set:
+
+Minimal Cut Set
+================
+
+The minimal cut set generation is a special mode where Kind 2 computes a minimal subset of
+the model elements (contract items, equations or node calls) such that the system become unsafe if we remove these assumptions.
```

Comment:
contract items = assumptions, guarantees
such that the system become unsafe if we remove these assumptions = whose violation leads the system to an unsafe state